src/openrct2/rct2/S6Importer.cpp

```diff
@@ -544,18 +544,73 @@ namespace RCT2
         {
             if (String::Equals(_s6.scenario_filename, "Europe - European Cultural Festival.SC6"))
             {
-                // This scenario breaks pathfinding. Create passages between the worlds. (List is grouped by neighbouring
-                // tiles.)
-                // clang-format off
+                // This scenario breaks pathfinding
                 FixLandOwnershipTilesWithOwnership(
                     {
-                        { 67, 94 }, { 68, 94 }, { 69, 94 },
-                        { 58, 24 }, { 58, 25 }, { 58, 26 }, { 58, 27 }, { 58, 28 }, { 58, 29 }, { 58, 30 }, { 58, 31 }, { 58, 32 },
-                        { 26, 44 }, { 26, 45 },
-                        { 32, 79 }, { 32, 80 }, { 32, 81 },
+                        { 67, 94 },
+                        { 68, 94 },
+                        { 69, 94 },
+                        { 58, 24 },
+                        { 58, 25 },
+                        { 58, 26 },
+                        { 58, 27 },
+                        { 58, 28 },
+                        { 58, 29 },
+                        { 58, 30 },
+                        { 58, 31 },
+                        { 58, 32 },
+                        { 26, 44 },
+                        { 26, 45 },
+                        { 32, 79 },
+                        { 32, 80 },
+                        { 32, 81 },
```

Comment:
Please revert. This was formatted this way deliberately. Notice how the coordinates on the same line are close together.

Reply:
Because the `// clang-format off/on` comments have been removed. They were there for these lists.